packages/tadviewer/src/components/CellClickData.tsx

```diff
@@ -0,0 +1,8 @@
+export interface CellClickData {
+  readonly value: string;
```

Comment:
Have you experimented with what gets passed as `value` for numeric or structured columns?  I don't think it will be passed to this callback as a `string` (but please check!).  I suspect `any` is probably the best we can do.

Reply:
Good point, `any` is more accurate. Updated!